### DISCUSSION TITLE
[GitHub → Eclipse] Can not import.

### DISCUSSION THREAD

**Discussion by bedcoding** [community/community#23744] [New to GitHub]
<p>Hello, I started Github from yesterday.</p>
<ol>
<li>
<p>First, I put the project in feather hub.</p>
</li>
<li>
<p>We also created two versions through the Branch function.<br>
     (Version: master1, master2)</p>
</li>
<li>
<p>Now is the master2 version status.<br>
     But I tried to go back to the master1 version, but I did not import it.</p>
</li>
</ol>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/9606e6bcafa3e6c4b4e0e847e336b12cea0fed95.jpeg" data-download-href="/uploads/short-url/lpcpcR1k34da4pjEIW9XEDi3KxT.jpg?dl=1" title="제목 없음0.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079178-63958df8-4094-4dea-bd8a-7a0658cd8326.jpg" alt="제목 없음0.jpg" data-base62-sha1="lpcpcR1k34da4pjEIW9XEDi3KxT" width="690" height="390" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/9606e6bcafa3e6c4b4e0e847e336b12cea0fed95_2_690x390.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/9606e6bcafa3e6c4b4e0e847e336b12cea0fed95_2_1035x585.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/9606e6bcafa3e6c4b4e0e847e336b12cea0fed95_2_1380x780.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/9606e6bcafa3e6c4b4e0e847e336b12cea0fed95_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음0.jpg</span><span class="informations">1920×1087 211 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>I proceeded like this.</p>
<p>Did I ever click on the wrong one?</p>
<p>  <img src="https://user-images.githubusercontent.com/33927558/181079179-680ee982-d49c-46e3-a425-ef72828b977a.jpeg" alt="제목 없음.jpg" data-base62-sha1="y3VdWQO3DqZUXT2NfFHvGKKON3D" width="543" height="451"></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e922bc60e3982d6dc7c3b77f80cc4caeecd60bd1.jpeg" data-download-href="/uploads/short-url/xgpFExE9DQ6LjvETnlH1s0MIYnL.jpg?dl=1" title="제목 없음2.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079180-f1c1959c-2317-4cf2-b817-427ae24908bb.jpeg" alt="제목 없음2.jpg" data-base62-sha1="xgpFExE9DQ6LjvETnlH1s0MIYnL" width="470" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e922bc60e3982d6dc7c3b77f80cc4caeecd60bd1_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음2.jpg</span><span class="informations">511×543 49.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/6ed33352e9237d17bdf34984459bdbc4c77730b4.jpeg" data-download-href="/uploads/short-url/fOp12bSVVBGkPxfjDj9TMdmX1u4.jpg?dl=1" title="제목 없음3.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079181-f9903b09-32e8-488f-9ff0-61704caa1aa2.jpeg" alt="제목 없음3.jpg" data-base62-sha1="fOp12bSVVBGkPxfjDj9TMdmX1u4" width="470" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/6ed33352e9237d17bdf34984459bdbc4c77730b4_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음3.jpg</span><span class="informations">511×543 32.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/25e4421d751792f5035ed69f5de0722d22c49294.jpeg" data-download-href="/uploads/short-url/5pcHRxGZOcs5bMfCpj085W9qKWg.jpg?dl=1" title="제목 없음4.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079182-e396fc0c-a96f-43a2-837a-1f7919bd8b6d.jpeg" alt="제목 없음4.jpg" data-base62-sha1="5pcHRxGZOcs5bMfCpj085W9qKWg" width="470" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/25e4421d751792f5035ed69f5de0722d22c49294_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음4.jpg</span><span class="informations">511×543 50 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/bbb201f8ce0678e522520044831bf785ce773fa7.jpeg" data-download-href="/uploads/short-url/qMqFkRB16DaXdsiNoFjGrk0IkyX.jpg?dl=1" title="제목 없음5.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079184-36f750f0-7726-4fd6-bc22-4bbe76090e0e.jpeg" alt="제목 없음5.jpg" data-base62-sha1="qMqFkRB16DaXdsiNoFjGrk0IkyX" width="470" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/bbb201f8ce0678e522520044831bf785ce773fa7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음5.jpg</span><span class="informations">511×543 43.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/e537ea16460befaa3acf61def0dfc1318b93eb49.jpeg" data-download-href="/uploads/short-url/wHL8tOgN1KoFpywy7Irh1ucqw7D.jpg?dl=1" title="제목 없음6.jpg" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/33927558/181079185-d23ed51f-afd2-4f93-9a5b-d2099d2d865f.jpeg" alt="제목 없음6.jpg" data-base62-sha1="wHL8tOgN1KoFpywy7Irh1ucqw7D" width="470" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/e537ea16460befaa3acf61def0dfc1318b93eb49_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">제목 없음6.jpg</span><span class="informations">511×543 42.4 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by Wabri** [comment, marked as answer]
<p>Eclipse have a strong plugins for git, where you can do allmost everything… So first of all open the relative prospective:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181079186-3178a70e-7ecb-4514-a8e6-2d8bd4f66e29.png" alt="first.png" data-base62-sha1="1KlHEPXGzYEjkFaMTIynus3GSma" width="643" height="345"></p>
<p>then select ‘git’ and open it:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181079187-32aaaa91-7a66-43fe-ad62-c5f443dc1525.png" alt="second.png" data-base62-sha1="dXyQz1JDrEhPUzVWIojCdJoSes2" width="399" height="396"></p>
<p>you can now see in the top right appear this (you can switch from views):</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181079189-258c7ba7-7b08-48db-9c3e-b8a4da71b389.png" alt="third.png" data-base62-sha1="1pveIk26vwvuM2rB3qPfCnvkrAZ" width="126" height="45"></p>
<p>in this view you can clone a remote repository by coping the link, using the box in the left called ‘Git Repositories’:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181079190-38667e26-6a12-4d58-9fb0-f73930d9551f.png" alt="fourth.png" data-base62-sha1="bMPQdYpjiel3m68zLVhXgfglib1" width="670" height="255"></p>
<p>then clone the repo and import that to your local worksapce:</p>
<p><img src="https://user-images.githubusercontent.com/12409541/181079192-f879db5a-97d7-43d4-976b-8ed51643ef1c.png" alt="six.png" data-base62-sha1="uZVc1S6r4ncKLWF92QSfbmNRpn0" width="669" height="497"></p>
<p>now you have your project opened. To navigate in the repository’s history you have to open history box by rigth click on the clone option:</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/2/208b26e135faf73996927bde626aa41b3cc26aed.png" data-download-href="/uploads/short-url/4DTpseamcuJsx5YT2Z4joUn3QOF.png?dl=1" title="five.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/12409541/181079193-c1f52c40-29f5-408f-8435-0f08d3498330.png" alt="five.png" data-base62-sha1="4DTpseamcuJsx5YT2Z4joUn3QOF" width="598" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/2/208b26e135faf73996927bde626aa41b3cc26aed_2_598x500.png, https://aws1.discourse-cdn.com/github/original/2X/2/208b26e135faf73996927bde626aa41b3cc26aed.png 1.5x, https://aws1.discourse-cdn.com/github/original/2X/2/208b26e135faf73996927bde626aa41b3cc26aed.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/2/208b26e135faf73996927bde626aa41b3cc26aed_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">five.png</span><span class="informations">693×579 52.7 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p>now you can see the special box ‘history’ with all commits, and you can change branch by checkout to your master2.</p>
<p>hope to solve your problem.</p>

**Comment by noahcil** [comment]
<p>What theme are you using for eclipse in these screenshots?</p>

**Comment by Wabri** [comment]
<p>Is darkest dark, you can find in the marketplace of eclipse:</p>
<p><a href="https://marketplace.eclipse.org/content/darkest-dark-theme-devstyle" rel="nofollow noopener">http://darkest-dark-theme-devstyle</a></p>

**Comment by Hrishikesh2900** [comment]
<p>Want to import Git project in eclipse?<br>
Here’s the best solution on the topic of importing Git project in eclipse</p>
<p>Just go through this link and you get a good solution foe the same!<br>
</p><aside class="onebox whitelistedgeneric">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/54510322/181079196-300f0eb5-45b8-4bb8-bb7f-0ef637e6f7ab.jpeg" class="site-icon" width="235" height="87">
      <a href="https://atechdaily.com/posts/Add-Existing-git-project-to-Eclipse" target="_blank" rel="nofollow noopener">atechdaily.com</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:235/87;"><img src="https://user-images.githubusercontent.com/54510322/181079197-3fd0a39c-4f55-4464-8a0c-11f1e2f63085.jpeg" class="thumbnail" width="235" height="87"></div>

<h3><a href="https://atechdaily.com/posts/Add-Existing-git-project-to-Eclipse" target="_blank" rel="nofollow noopener">Import Git project in Eclipse</a></h3>

<p>Many users have problem importing a git project to Eclipse. So lets learn how to do it by following these steps.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
